### PR TITLE
added migration script for sales invoice

### PR DESCRIPTION
**Pull request by lakshgoel95**

### CODE REVIEW

File: src/server/db/migrations/23-permission23-1.main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const path = require('path');
+
+module.exports.up = function(db,config) {
+	const permissions = require('../data/permission-23.1.json');
+	return db.queryInterface.bulkInsert('Permission',permissions);
+}
+
+module.exports.down = function(db, config) {
+	const permissions = require('../data/permission-23.1.json');
+	permissions.map(perm => {
```
Comment (kwierchris) [inline src/server/db/migrations/23-permission23-1.main.js]: No promise returned.

File: src/server/db/migrations/23-permission23-1.main.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		return db.queryInterface.bulkDelete('Permisssion', {
+			authority:[perm.authorityId],
```
Comment (kwierchris) [inline src/server/db/migrations/23-permission23-1.main.js]: AFAIK bulkDelete requires an array of objects.

File: src/server/db/migrations/23-permission23-1.main.js (hunk 1 above)
Comment (lakshgoel95) [reply src/server/db/migrations/23-permission23-1.main.js]: I added the return statement and pushed the code.

File: src/server/db/migrations/23-permission23-1.main.js (hunk 2 above)
Comment (lakshgoel95) [reply src/server/db/migrations/23-permission23-1.main.js]: I checked the bulkDelete function and it accepts only object. The bulkInsert function requires array of objects.

File: src/server/db/migrations/23-permission23-1.main.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+const path = require('path');
+
+module.exports.up = function(db,config) {
+	const permissions = require('../data/permission-23.1.json');
+	return db.queryInterface.bulkInsert('Permission',permissions);
+}
+
+module.exports.down = function(db, config) {
+	const permissions = require('../data/permission-23.1.json');
+	return permissions.map(perm => {
```
Comment (kwierchris) [inline src/server/db/migrations/23-permission23-1.main.js]: This is an array of promises but we need a single promise.

**Pull request merged**